src/Routes/Index.js

```diff
@@ -43,6 +44,96 @@ class IndexRoute extends BaseRoute {
+                res.render('error', { title: 'Success', status: 200, message: 'Section has been added.' });
+            } catch (e) {
+                handleError(this.db, req, res, e.stack);
+            }
+        });
+
+        this.router.get('/about/manage/:id', this.requiresAuth.bind(this), this.isAdmin.bind(this), (req, res) => {
+            this.db.select().from('about').where({ id: req.params.id }).then((data) => {
+                if (!data.length) return res.status(404).render('error', {
+                    title: 'Page not found',
+                    status: 404,
+                    message: 'The page you were looking for could not be found.'
+                });
+                res.render('about/edit', { title: 'Edit Section', data: data[0] });
+            }).catch((e) => {
+                handleError(this.db, req, res, e.stack);
+            });
+        });
+
+        this.router.post('/about/manage/:id', this.requiresAuth.bind(this), this.isAdmin.bind(this), (req, res) => {
+            this.db.select().from('about').where({ id: req.params.id }).then(async(data) => {
+                if (!data.length) return res.status(404).render('error', {
+                    title: 'Page not found',
+                    status: 404,
+                    message: 'The page you were looking for could not be found.'
+                });
+                let changes = {};
+                const validate = FormValidator.validateAboutSection(req.body);
+                if (validate && validate.length > 0) return res.render('about/edit', { title: 'Edit Section', data: req.body, errors: validate });
+                const columns = Object.keys(await this.db('about').columnInfo());
+                for (const column of columns) {
+                    if (req.body[column]) {
+                        changes[column] = req.body[column];
+                    } else {
+                        changes[column] = null;
+                    }
+                }
+                await this.db('about').where({ id: req.body.id }).update(changes);
+                res.render('error', { title: 'Success', status: 200, message: 'Section has been updated.' });
+            }).catch((e) => {
+                handleError(this.db, req, res, e.stack);
+            });
+        });
+
+        this.router.get('/about/manage/:id/delete', this.requiresAuth.bind(this), this.isAdmin.bind(this), (req, res) => {
+            this.db.select().from('about').where({ id: req.params.id }).then(async(data) => {
+                if (!data.length) return res.status(404).render('error', {
+                    title: 'Page not found',
+                    status: 404,
+                    message: 'The page you were looking for could not be found.'
+                });
+                await this.db('about').where({ id: req.params.id }).del();
+                res.render('error', { title: 'Success', status: 200, message: 'Section has been deleted.' });
```

Comment:
Redirect back to the manage page here?

Seems like an annoying flow to remove a section and then end up on the error page, with the only CTA being to view lists...